README.md

```diff
@@ -1,48 +1,117 @@
 )](https://ci.appveyor.com/project/Xablu/xablu-webapiclient)
 ![GitHub tag](https://img.shields.io/github/tag/Xablu/Xablu.WebApiClient.svg)
 [![NuGet](https://img.shields.io/nuget/v/Xablu.WebApiClient.svg?label=NuGet)](https://www.nuget.org/packages/Xablu.WebApiClient/)
 [![MyGet](https://img.shields.io/myget/xabluhq/v/Xablu.WebApiClient.svg)](https://www.myget.org/F/Xablu.WebApiClient/api/v2)
 
-## Usage
+## Key Features <a name="features"></a>
 
-Before using the Xablu.WebApiClient plugin (via `CrossRestApiClient.Current`) make sure you configure the Xablu.WebApiClient plugin with at least the base address of the API you want to connect to.
-Failing to do so will result in a `NotConfiguredException` when using the plugin.
+The WebApiClient contains new features with respect to the previous version. The list of key features is depicted below: 
 
-> Note: it is no longer needed to supply a platform specific `HttpMessageHandler`, this is now build in into the plugin. This means that the configuration (as shown below) can all be done in 
-> your shared code.
+REST Client:
+  * Based on Refit                                          ✔
+  * Implemented Retry and Timeout from Polly                ✔
+  * Implemented Fusillade’s Priorities                      ✔
 
-Configuration is done through the `CrossRestApiClient.Configure` method, like this:
+GraphQL Client: 
+  * Based on GraphQL.Client                                 ✔
+  * Implemented Retry and Timeout from Polly                ✔
+  * Implemented Fusillade’s Priorities                      ✔
+  * Implemented a Custom, Object-Oriented Query Builder     ✔
 
-```
-CrossRestApiClient.Configure((opt) => opt.ApiBaseAddress = "https://api.mybackend.com");
-```
+## Example <a name="examples"></a>
 
-Once you have configured the Xablu.WebApiClient plugin you can start using it through the static `CrossRestApiClient.Current` property, like this:
+Make sure to check out more examples in the Console Test App in the package. Down here is an example call for connecting with a Web API service through Refit:
 
+Abstract:
 ```
-var client = CrossRestApiClient.Current;
-var result = await client.GetAsync<IEnumerable<TodoItem>>(Priority.UserInitiated, "mypath").ConfigureAwait(false);
-
-if(!result.IsSuccessStatusCode)
+Task<TResult> Call<TResult>(Func<T, Task<TResult>> operation, Priority priority, int retryCount, Func<Exception, bool> shouldRetry, int timeout); 
+```
+Example implementation:
+```
+async Task<IEnumerable<MyModel>> GetModelsItemsAsync(bool forceRefresh = false) 
 {
-  // Handle error response...
+  IWebApiClient<IRefitInteraface> webApiClient = WebApiClientFactory.Get<IRefitInteraface>("baseURL", bool defaultHeaders: true);
+  var jsonresult = await webApiClient.Call(
+      (IRefitInterafaceService) => IRefitInterafaceService.GetTask(),
+      (Polly.Priority) Priority.UserInitiated,
+      (retryCount) 2,
+      (shouldRetry) => true,
+      (timeout) 60); 
```

Comment:
Shouldn't parameter names be set like this: `timeout: 60` instead? Also the first one, which is a parameter for a `Func` should start with lowercase (something like `(myRefitService) => myRefitService.GetData()`)

Reply:
done